pkg/api/service/testing/make.go

```diff
@@ -99,6 +101,18 @@ func SetAllocateLBNodePortFalse(svc *api.Service) {
 	svc.Spec.AllocateLoadBalancerNodePorts = utilpointer.BoolPtr(false)
 }
 
+// SetInternalTrafficPolicyCluster sets the internalTrafficPolicy field to "Cluster"
+func SetInternalTrafficPolicyCluster(svc *api.Service) {
```

Comment:
Also, please put these at the end of the file - I tried to keep it roughly sorted by "most likely to be used"

Reply:
updated